components/script_bindings/webidls/ReportBody.webidl

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// https://w3c.github.io/reporting/#interface-reporting-observer
+// TODO: Rename file to `ReportingObserver.webidl` with the other interfaces
+
+[Exposed=(Window,Worker)]
+interface ReportBody {
+  [Default] object toJSON();
+};
```

Comment:
Curious, the method is not defined in https://w3c.github.io/reporting/#interface-reporting-observer.
MDN says it should exist but links to a section of the spec that no longer exists: https://developer.mozilla.org/en-US/docs/Web/API/ReportBody/toJSON#specifications.